tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1571,6 +1582,33 @@ def test_copy_dest_lock(ansible_zos_module):
         hosts.all.zos_data_set(name=DATASET_2, state="absent")
 
 
+@pytest.mark.uss
+@pytest.mark.parametrize("src", [
+    dict(src="/etc/", is_remote=False),
+    dict(src="/etc/", is_remote=True),])
+def test_ensure_copy_directory_does_not_change_permission_on_dest(ansible_zos_module, src):
```

Comment:
Nice to see a test case added but it seems unrelated to the git issue, i just wonder where it came from, was it intentionally added, trying to understand how I should evaluate this.